src/thrift_0_13_0/Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+THRIFT_VERSION = 0.13.0
+THRIFT_VERSION_FULL = $(THRIFT_VERSION)-6
+
+MAIN_TARGET = libthrift-$(THRIFT_VERSION)_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = libthrift-dev_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb \
+		  python3-thrift_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb \
+		  thrift-compiler_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+
+THRIFT_LINK_PRE = http://deb.debian.org/debian/pool/main/t/thrift/
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -rf thrift-$(THRIFT_VERSION)
+
+	wget "$(THRIFT_LINK_PRE)/thrift_$(THRIFT_VERSION_FULL).debian.tar.xz"
```

Comment:
>wget [](http://example.com/codeflow?start=1&length=4)

If you run `make` twice, 2nd time wget will rename because name conflicts with local.

Reply:
`wget -O` will be better than `rm` a wildcard.

Reply:
Ok